ssh/DOCS.md

```diff
@@ -249,6 +249,29 @@ based on the following:
 - `MINOR`: Backwards-compatible new features and enhancements.
 - `PATCH`: Backwards-compatible bugfixes and package updates.
 
+## Visual Studio Code Remote - SSH
+
+Some customization is required prior to being able to connect to your Home 
+Assistant install with Visual Studio Code. The following configuration will aid
+in setting up VSCode Remote - SSH:
+
+```yaml
+packages:
+  - gcompat
+  - libstdc++
+  - curl
+allow_remote_port_forwarding: true
+allow_tcp_forwarding: true
+```
+
+### Notes
+* Allowing remote port and tcp forwarding lower the overall level of security of 
+your SSH server. Nevertheless, this warning is debatable. 
+* It is *not* required to disable protection mode for this to work correctly.
```

Comment:
Unrelated comment. It is also not required to turn on your kitchen light... Let's remove it.

Reply:
This will be an inevitable question when someone with a weird setup does the suggested config and then doesn't get a working outcome.